CONTRIBUTING.md

```diff
@@ -323,44 +325,27 @@ You may run `minikube start --memory 8192` to start Minikube with a 8GiB
 VM underneath.
 
 
-
-
-## Running tests
-
-This git repository contains `pytest` based tests that you can run locally.
-Depending on your development setup, you may want to run different kind of
-tests. You can get some hints on what tests to run and how by inspecting
-`.travis.yaml`.
-
-### Environment variables influencing tests
-- `BINDER_URL`: An address of an already running BinderHub as reachable from the
-  tests. If this is set, the test suite will not start temporary local BinderHub
-  servers but instead interact with the remote BinderHub.
-- `GITHUB_ACCESS_TOKEN`: A GitHub access token to help avoid quota limitations
-  for anonymous users. It is used to enable certain tests making many calls to
-  GitHub API.
-
-### Pytest marks labelling tests
-- `remote`: Tests for them the BinderHub is already running somewhere.
-- `github_api`: Tests that communicate with the GitHub API a lot.
-- `auth`: Tests related to BinderHub's usage of JupyterHub as an OAuth2 Identity
-  Provider (IdP) for non public access.
-
-
```

Comment:
The overview this section contain took quite some time for me to figure out by code inspection and has been important to me to comprehend - can we retain it?

Reply:
I was thinking about having a dedicated section in the docs that contains all the nitty gritty details, how to run all the tests, other pitfalls, etc. With the goal of keeping `CONTRIBUTING.md` aimed at newcomers or a quick reference.

Having such a place to store commands I actually use (because CONTRIBUTING.md uses "a Ubuntu system as common denominator" would be nice.

Reply:
Sure go for it!